client.go

```diff
@@ -362,213 +108,152 @@ func (client *Client) AddWithRetention(key string, timestamp int64, value float6
-	if err != nil {
-		return nil, err
-	}
-	ranges, err = ParseRanges(info)
-	return ranges, err
+	mrangeOptions := NewMultiRangeOptions()
+	mrangeOptions = mrangeOptions.SetAggregation(aggType, bucketSizeSec)
+	return client.MultiRangeWithOptions(fromTimestamp, toTimestamp, *mrangeOptions, filters...)
 }
 
 // MultiRangeWithOptions - Query a timestamp range across multiple time-series by filters.
 // args:
-// fromTimestamp - start of range
-// toTimestamp - end of range
+// fromTimestamp - start of range. You can use TimeRangeMinimum to express the minimum possible timestamp.
+// toTimestamp - end of range. You can use TimeRangeFull or TimeRangeMaximum to express the maximum possible timestamp.
 // mrangeOptions - MultiRangeOptions options. You can use the default DefaultMultiRangeOptions
 // filters - list of filters e.g. "a=bb", "b!=aa"
 func (client *Client) MultiRangeWithOptions(fromTimestamp int64, toTimestamp int64, mrangeOptions MultiRangeOptions, filters ...string) (ranges []Range, err error) {
 	conn := client.Pool.Get()
 	defer conn.Close()
-
+	var reply interface{}
 	args := createMultiRangeCmdArguments(fromTimestamp, toTimestamp, mrangeOptions, filters)
-
-	info, err := conn.Do("TS.MRANGE", args...)
+	reply, err = conn.Do("TS.MRANGE", args...)
 	if err != nil {
 		return
 	}
-	ranges, err = ParseRanges(info)
+	ranges, err = ParseRanges(reply)
 	return
 }
 
-func createMultiRangeCmdArguments(fromTimestamp int64, toTimestamp int64, mrangeOptions MultiRangeOptions, filters []string) []interface{} {
-	args := []interface{}{strconv.FormatInt(fromTimestamp, 10), strconv.FormatInt(toTimestamp, 10)}
-	if mrangeOptions.AggType != -1 {
-		args = append(args, "AGGREGATION", mrangeOptions.AggType.String(), strconv.Itoa(mrangeOptions.TimeBucket))
-	}
-	if mrangeOptions.Count != -1 {
-		args = append(args, "COUNT", strconv.FormatInt(mrangeOptions.Count, 10))
+// Get - Get the last sample of a time-series.
+// args:
+// key - time-series key name
+func (client *Client) Get(key string) (dataPoint *DataPoint,
+	err error) {
+	conn := client.Pool.Get()
+	defer conn.Close()
+	resp, err := conn.Do("TS.GET", key)
+	if err != nil {
+		return nil, err
 	}
-	if mrangeOptions.WithLabels == true {
-		args = append(args, "WITHLABELS")
+	dataPoint, err = ParseDataPoint(resp)
+	return
+}
+
+// MultiGet - Get the last sample across multiple time-series, matching the specific filters.
+// args:
+// filters - list of filters e.g. "a=bb", "b!=aa"
+func (client *Client) MultiGet(filters ...string) (ranges []Range,
+	err error) {
+	conn := client.Pool.Get()
+	defer conn.Close()
+	var reply interface{}
+	if len(filters) == 0 {
+		return
 	}
-	args = append(args, "FILTER")
+	args := []interface{}{"FILTER"}
 	for _, filter := range filters {
 		args = append(args, filter)
 	}
-	return args
+
+	reply, err = conn.Do("TS.MGET", args...)
+
+	if err != nil {
+		return
+	}
+	ranges, err = ParseRangesSingleDataPoint(reply)
+	return
+}
+
+// Info create a new time-series
```

Comment:
Wrong description

Reply:
Fixing on new commit and revising all methods description